packages/rest-client-jsonapi/src/services/utils/query/query.test.ts

```diff
@@ -57,21 +57,23 @@ describe("rest-client-jsonapi/services/utils/query", () => {
 
   describe("fieldsToQueryParam", () => {
     it("works", () => {
-      expect(() =>
+      // TODO need to handle namespace.field. Jira link: https://bitovi.atlassian.net/browse/HATCH-387
```

Comment:
@arthurpankiewicz per commenting out the code that throws this error in query.ts, I assume it's fine to comment this out for now as well?

Reply:
yup will address soon after